src/components/EnvironmentOverride/SecretOverrides.tsx

```diff
@@ -269,14 +246,26 @@ export function OverrideSecretForm({ name, appChartRef, toggleCollapse }) {
         PATTERNS.CONFIG_MAP_AND_SECRET_KEY,
         `Key must consist of alphanumeric characters, '.', '-' and '_'`,
     )
+    const isEsoSecretData = esoSecretData?.secretStore && esoSecretData?.esoData.length > 0
     const [yamlMode, toggleYamlMode] = useState(true)
+    const [esoDataSecret, setEsoData] = useState(esoSecretData?.esoData)
+    const [secretStore, setSecretStore] = useState(esoSecretData?.secretStore)
     const [isFilePermissionChecked, setIsFilePermissionChecked] = useState(!!filePermission)
+    const [esoSecretYaml, setEsoYaml] = useState(YAML.stringify(isEsoSecretData ? esoSecretData : {}))
+    const sample = YAML.stringify(sampleJSONs[externalType] || sampleJSONs["default"])
```

Comment:
Please use ' in tsx i.e. "default" to 'default'

Reply:
Done